modules/op-pac-additional-controller.adoc

```diff
@@ -0,0 +1,38 @@
+// This module is included in the following assemblies:
+// * pac/install-config-pipelines-as-code.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="pac-additional-controller_{context}"]
+= Configuring additional {pac} controllers to support additional GitHub apps
+
+[role="_abstract"]
+By default, you can configure {pac} to interact with one GitHub app. In some cases you might need to use more than one GitHub app, for example, if you need to use different GitHub accounts or different GitHub instances such asGitHub Enterprise or GitHub SaaS. If you want to use more than one GitHub app, you must configure an additional {pac} controller for every additional GitHub app.
+
+.Procedure
+
+. In the `TektonConfig` custom resource, in the `platforms.openshift.pipelinesAsCode` spec, add the `additionalPACControllers` section, as in the following example:
++
+.Example `additionalPACControllers` section
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  platforms:
+    openshift:
+      pipelinesAsCode:
+        additionalPACControllers:
+          pac_controller_2:  # <1>
+            enable: true    # <2>
+            secretName: pac_secret_2  # <3>
+            settings: #  <4>
+# ...
+----
+<1> The name of the controller. This name must be unique and not exceed 25 characters in length.
+<2> This parameter is optional. Set this parameter to `true` to enable the additional controller or to `false` to disable the additional controller. The default vaule is `true`.
+<3> Set this parameter to the name of a secret that you must create for the GitHub app.
+<4> This section is optional. In this section, you can set any {pac} settings for this controller if the settings must be different from the main {pac} controller.
+
+. Optional: If you want to use more than two GitHub apps, create addititonal sections under the `pipelinesAsCode.additionalPACCControllers` spec to configure a {pac} controller for every GitHub instance. Use a unique name for every controller.
```

Comment:
I think this reads fine as is, but you could use the first phrase of the sentence to introduce the condition with a colon:
```suggestion
. If you want to use more than two GitHub apps: Create addititonal sections under the `pipelinesAsCode.additionalPACCControllers` spec to configure a {pac} controller for every GitHub instance. Use a unique name for every controller.
```

Up to you though, either way seems good